src/components/list-lemplate/list-template.tsx

```diff
@@ -1,10 +1,12 @@
 import { Dispatch, SetStateAction } from "react";
-import { Department, Skill } from "cv-graphql";
+import { Department, Skill, User, UserRole } from "cv-graphql";
 import { Button, Input } from "antd";
 import { t } from "i18next";
```

Comment:
you should get { t } from useTranslation hook if you want your component to automatically re-render on language change

Reply:
fixed